frontend/ui/quickstart.lua

```diff
@@ -10,61 +10,185 @@ local lfs = require("libs/libkoreader-lfs")
+* [INSTALLING DICTIONARIES](#dicts)
+* [MORE INFO](#more)
+
+---
+
+You can access the complete user manual from [our GitHub page](https://github.com/koreader/koreader).
+
+## USER INTERFACE <a id="ui"></a>
+
+<div class="img-block">![Touch zones](../resources/quickstart/touchzones.png)</div>
+
+- To show the **TOP MENU** or **BOTTOM MENU** you can click the indicated zones. You can click or swipe down the upper zone to show the **TOP MENU**.
+
+- **STATUS BAR** zone can be used to cycle between STATUS BAR items if one item is visible. This will also hide and show the STATUS BAR if you tap enough times.
+
+## USER INTERFACE TIPS <a id="uitips"></a>
+
+- You can change interface language via:
+
+> **TOP MENU ➔ ![Settings](../resources/icons/mdlight/appbar.settings.svg) ➔ Language**
+
+- If you tap and hold an option or menu item (font weight, line spacing etc.), you can set its value as **DEFAULT**.  The new value will only be applied to documents opened from now on. Previously opened documents will keep their settings. You can identify default values as a STAR in menu or as a black border around indicators as seen below:
+
+<div class="img-block break-before-avoid">![Default setting 1](../resources/quickstart/defaultsetting1.png)</div>
+<div class="img-block break-before-avoid">![Default setting 2](../resources/quickstart/defaultsetting2.png)</div>
+
+- You can see explanations for all items on the **BOTTOM MENU** by tapping and holding the option’s name. This is also available for most of the **TOP MENU** menu items.
+
+- You can **CLOSE** full screen dialogs (History, Table of Contents, Bookmarks, Reading Statistics etc.) by swiping down
+
+- **SCREENSHOTS** can be taken by touching opposing corners of the screen diagonally at the same time or by making a long diagonal swipe
+
+<div class="img-block break-after-avoid">![Number picker](../resources/quickstart/numberpicker.png)</div>
+
+- In dialogs containing adjustment arrow buttons like the one above, you can tap and hold on arrow buttons to increase / decrease the value in bigger increments
+
+- You can **CLOSE** this type of dialog (non-full screen) by tapping outside of the window. You can **MOVE** this type of dialog by holding the window title and dragging
+
+- You can make this type of dialog **SEMI-TRANSPARENT** (to see the text under it while adjusting a value) by tapping and holding the window title
+
+- Tapping and holding a word brings up a dialog where you can also search the selection in the document for more occurrences or look it up on Wikipedia
+
+- You can highlight sections by tapping and holding a word and dragging your finger
+
+- You can move through your document via **SKIM DOCUMENT** dialog:
+
+> **TOP MENU ➔ ![Navigation](../resources/icons/mdlight/appbar.navigation.svg) ➔ Skim document**
+
+## ACCESSING FILES <a id="afiles"></a>
+
+The following methods are available for accessing your books and articles:
+
+- File Browser
+- Favorites
+- History
```

Comment:
Actually, does the MD treat `*` and `-` identically? If so, please keep to `*`. :-)
```suggestion
- File Browser
- Favorites
- History
```